Loads/ILoads/IPoint2dForce.cs

```diff
@@ -0,0 +1,10 @@
+using OasysUnits;
+
+namespace MagmaWorks.Taxonomy.Loads
+{
+    public interface IPoint2dForce : ILoad
```

Comment:
The force is the 2d vector, that could also be applied on a 3d point..?

Reply:
What i mean is, would `IPointForce2d` not be more correct? ;)

Reply:
Changed in https://github.com/MagmaWorks/Taxonomy/pull/32/commits/31477d982652a44e8c37e8b67b1a2aeb8b1e57a5